pmm-app/src/pmm-pt-summary/datasource/PTSummary.service.ts

```diff
@@ -0,0 +1,7 @@
+import { apiRequestManagement } from 'shared/components/helpers/api';
+
+export const PTSummaryService = {
+  async getPTSummary(body) {
+    return apiRequestManagement.post<any, any>('/Actions/StartPTSummary', body);
```

Comment:
Can't change the` <any, any>` because of the `<void | T>` in the `ApiRequest`. I don't understand why `void` is there and I don't think makes sense to add it to this types (e.g. `<Response | void, Request>`)

Reply:
What do you think @atymchuk ?